commercetools/resource_cart_discount_absolute_test.go

```diff
@@ -0,0 +1,86 @@
+package commercetools
```

Comment:
Can we also add a test for the fixed discount? That way we have all cases except the `giftLineItem` covered